components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -209,32 +209,16 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
 
         let leaveChannel;
         if (channel.name !== Constants.DEFAULT_CHANNEL) {
-            if (channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) {
-                leaveChannel =
+            leaveChannel =
                     (<Menu.Group>
                         <Menu.ItemAction
                             id={`leave-${channel.id}`}
                             onClick={this.handleLeaveChannel}
                             icon={<i className='icon-close'/>}
                             text={leaveChannelText}
+                            isDangerous={(channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) === false}
```

Comment:
no need to check for `=== false` ... you can just revert the boolean by using `!` like this:

```suggestion
                            isDangerous={!(channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL)}
```